client/makelunch.js

```diff
@@ -32,7 +42,7 @@ Meteor.startup(function () {
       path:'/editmeal/:_id',
       data: function () {
         var meal = Meals.findOne(this.params._id)
-        console.log(meal)
+        if (!meal) return false
```

Comment:
Is this just a hot-code reload thing?  If so, I don't think we should be trying to handle it as it's not really relevant to production.  Generally, I'm all for predictable object structures, which isn't really what we've got here.